src/components/VContentSwitcher/VSearchTypeItem.vue

```diff
@@ -5,9 +5,9 @@
     v-bind="component"
     @click.native="$emit('click', item)"
   >
-    <div class="flex flex-row items-center text-base">
-      <VIcon :icon-path="icon" class="me-2 my-2" />
-      <span class="pe-20 font-semibold">{{ $t(`search-type.${item}`) }}</span>
+    <div class="flex flex-row items-center text-base gap-2 py-2">
```

Comment:
I think some of the changes here caused the 'Beta' pill to be close to the text instead of right-aligned.

Reply:
Fixed!